### PR TITLE
Reorganizing + splitting UX component libraries

### PR DESCRIPTION
**Pull request by Zee2**
## Overview
Up until this point, our RectTransform-based (preferred) UX components were jumbled in with our non-RectTransform-based (static) controls, which caused confusion + frustration, like #10741 .

This PR splits the component library into two packages, `com.microsoft.mrtk.uxcomponents` and `com.microsoft.mrtk.uxcomponents.static`. The naming of these is very much so up for debate, please let me know if you have some other preferred naming scheme. The ideas we went through:

- ❌ Call each of them "canvas" vs "non-canvas"
    - No, doesn't make it clear which ones we actually want people to use
- ❌ Deprecate them completely
    - No, at least not yet, they're better for perf in some situations, so having the option is nice
- ❌ Keep them together
    - No, very confusing, prefab soup, causes headaches as described in #10741 
- ❌ Call them "legacy"
    - Meh, they still work "fine", we just don't recommend them
- ❌ Call the old style "static"
    - ~Descriptive and good imo. The key difference in capability is that our preferred component library is dynamically laid out, flexible, responsive, resizable, etc... the old style is not. So, I feel like calling the old style "static" is really the clearest description!~ Team disagreed...
- ✔ Call the old style "non-canvas", call the new style... just uxcomponents
    - Simplest option. Team has decided to go forward with this one!

## Changes
- Created `com.microsoft.mrtk.uxcomponents.noncanvas`
- Moved all non-canvas UX controls to `com.microsoft.mrtk.uxcomponents.noncanvas`
- Reorganized some folders for clarity

## Note

Our automated tests only were using the non-Canvas components. So, our existing UX tests were migrated to the new static components package. TODO in the future, write equivalent Canvas tests and put them in the main UX package.

Also, there still needs to be some work done renaming things.

### CODE REVIEW

File: UnityProjects/MRTKDevTemplate/Packages/manifest.json [hunk 1]
```diff
@@ -15,7 +15,7 @@
     "com.microsoft.mrtk.spatialmanipulation": "file:../../../com.microsoft.mrtk.spatialmanipulation",
     "com.microsoft.mrtk.standardassets": "file:../../../com.microsoft.mrtk.standardassets",
     "com.microsoft.mrtk.tools": "file:../../../com.microsoft.mrtk.tools",
-    "com.microsoft.mrtk.uxcomponents": "file:../../../com.microsoft.mrtk.uxcomponents",
+    "com.microsoft.mrtk.uxcomponents.static": "file:C:/Users/finnn/src/MixedRealityToolkit-Unity-TENTED/com.microsoft.mrtk.uxcomponents.static",
```
Comment (keveleigh) [inline UnityProjects/MRTKDevTemplate/Packages/manifest.json:18]: Absolute paths 👀

Comment (Zee2) [reply UnityProjects/MRTKDevTemplate/Packages/manifest.json:18]: Ope, I keep doing this

Comment (Zee2) [reply UnityProjects/MRTKDevTemplate/Packages/manifest.json]: Fixed

File: UnityProjects/MRTKDevTemplate/Assets/Scripts/SystemKeyboardExample.cs [hunk 2]
```diff
@@ -1,163 +1,163 @@
-﻿// Copyright (c) Microsoft Corporation.
-// Licensed under the MIT License.
+﻿// // Copyright (c) Microsoft Corporation.
```
Comment (keveleigh) [inline UnityProjects/MRTKDevTemplate/Assets/Scripts/SystemKeyboardExample.cs:1]: Also, was this intentional?

Comment (Zee2) [reply UnityProjects/MRTKDevTemplate/Assets/Scripts/SystemKeyboardExample.cs:1]: No, was just testing some packaging things, didn't mean to commit

Comment (Zee2) [reply UnityProjects/MRTKDevTemplate/Assets/Scripts/SystemKeyboardExample.cs]: Fixed

**Review by davidkline-ms — APPROVED** [review]

**Comment by Zee2** [comment]
@davidkline-ms @MaxWang-MS , it stale'd the review 😅

**Review by davidkline-ms — APPROVED** [review]

**Pull request merged**